generated/nidaqmx/system/physical_channel.py

```diff
@@ -150,44 +98,26 @@ def ai_power_control_enable(self, val):
                 if cfunc.argtypes is None:
                     cfunc.argtypes = [
                         ctypes_byte_str, c_bool32]
-
         error_code = cfunc(
             self._name, val)
         check_for_error(error_code)
 
     @ai_power_control_enable.deleter
     def ai_power_control_enable(self):
-        cfunc = lib_importer.windll.DAQmxResetPhysicalChanAIPowerControlEnable
-        if cfunc.argtypes is None:
-            with cfunc.arglock:
-                if cfunc.argtypes is None:
-                    cfunc.argtypes = [
-                        ctypes_byte_str]
-
-        error_code = cfunc(
-            self._name)
-        check_for_error(error_code)
+        self._interpreter.reset_physical_chan_attribute(
+                self._name, 0x316d)
```

Comment:
The interpreter does not have a `reset_physical_chan_attribute` method.

Reply:
Sure. This has been handled in #302